autoprotocol/container_type.py

```diff
@@ -1238,6 +1238,84 @@ def row_count(self):
     cat_no="136101",
 )
 
+#:
+FLAT384BLACKUBOTTOM = ContainerType(
+    name="384-well Black Clear Round Bottom Ultra-low Attachment Microplate",
+    shortname="384-ubottom-black-clear-tc",
+    is_tube=False,
+    well_coating=None,
+    sterile=True,
+    well_count=384,
+    col_count=24,
+    well_depth_mm=12.55,
+    well_volume_ul=Unit(90, "microliter"),
+    dead_volume_ul=Unit(10, "microliter"),
+    safe_min_volume_ul=Unit(20, "microliter"),
+    capabilities=[
+        "pipette",
+        "spin",
+        "incubate",
+        "cover",
+        "stamp",
+        "dispense",
+        "dispense-destination",
+        "echo_dest",
+        "provision",
+        "uncover",
+        "seal",
+        "unseal",
+        "liquid_handle",
+        "absorbance",
+        "fluorescence",
+        "luminescence",
+        "bluewash",
+        "spectophotometry",
+    ],
+    cover_types=["universal"],
+    seal_types=["ultra-clear", "foil"],
+    vendor="Corning",
+    cat_no="3830",
+)
+
+#:
+FLAT384BLACKTC = ContainerType(
+    name="384-well Black Clear Flat Bottom Tissue Culture Microplate",
+    shortname="384-flat-black-clear-tc",
+    is_tube=False,
+    well_coating=None,
+    sterile=True,
+    well_count=384,
+    col_count=24,
+    well_depth_mm=11.55,
```

Comment:
well_depth_mm = Unit(11.55, "millimeter"),

Reply:
It is actually asking a float in the signature. 
https://github.com/autoprotocol/autoprotocol-python/blob/master/autoprotocol/container_type.py#L54